src/helm/benchmark/scenarios/legal_summarization_scenario.py

```diff
@@ -0,0 +1,194 @@
+from pathlib import Path
+
+from typing import List, Optional, Any
+
+import datasets
+from datasets import load_dataset
+
+from .scenario import Scenario, Instance, Reference, TRAIN_SPLIT, VALID_SPLIT, TEST_SPLIT, CORRECT_TAG, Input, Output
+
+_ALL_LANGUAGES = {
+    "bulgarian": "bg",
+    "czech": "cs",
+    "dutch": "nl",
+    "estonian": "et",
+    "french": "fr",
+    "greek": "el",
+    "irish": "ga",
+    "latvian": "lv",
+    "maltese": "mt",
+    "portuguese": "pt",
+    "slovak": "sk",
+    "spanish": "es",
+    "croatian": "hr",
+    "danish": "da",
+    "english": "en",
+    "finnish": "fi",
+    "german": "de",
+    "hungarian": "hu",
+    "italian": "it",
+    "lithuanian": "lt",
+    "polish": "pl",
+    "romanian": "ro",
+    "slovenian": "sl",
+    "swedish": "sv",
+}
+
+# IMPORTANT: So far, we only support English, since we would need to adapt the metrics for scoring the other languages.
+_SUPPORTED_LANGUAGES = {
+    "english": "en",
+}
+
+
+def _clean_and_truncate(text: str, max_length: Optional[int] = None) -> str:
+    text = text.replace("\n", " ")
+    whitespace_tokens = text.split()
+    if max_length:
+        whitespace_tokens = whitespace_tokens[:max_length]
+    return " ".join(whitespace_tokens)
+
+
+class LegalSummarizationScenario(Scenario):
+    """
+    Scenario for single document text summarization.
+    Currently, supports the following datasets:
+    1. BillSum (https://aclanthology.org/D19-5406/)
+    2. MultiLexSum (https://arxiv.org/abs/2206.10883)
+    3. EurLexSum (https://arxiv.org/abs/2210.13448)
+
+    Task prompt structure
+
+        Summarize the given document.
+        Document: {tok_1 ... tok_n}
```

Comment:
Is this {} actually part of the text?

Reply:
No. I think I copied this from the summarization scenario